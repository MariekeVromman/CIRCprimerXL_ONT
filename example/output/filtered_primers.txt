circ_ID	chr	start	end	primer_ID	FWD_primer	REV_primer	FWD_pos	FWD_length	REV_pos	REV_length	FWD_Tm	REV_Tm	FWD_GC	REV_GC	amplicon	PASS	start_annotation	end_annotation	splicing
circ2	chr1	17525	18061	4	CTTGGCACTGGAGAACAC	GTCTTCCCAGCAACATCAG	129	18	287	19	59.320	59.531	55.556	52.632	CTTGGCACTGGAGAACACCTGTGTCTGATGCCCTGGGTCCCCACTAAGCCAGGCCGGGCCTCCCGCCCACACCCCTCGGCCCTGCCCTCTGGCCATACAGGTTCTCGGTGGTGTTGAAGAGCAGCAAGGAGCTGACAGAGCTGATGTTGCTGGGAAGAC	PASS	exon_chr1:17914-18061	exon_chr1:17605-17675	spliced
circ1	chr1	17232	17742	3	TCCCTCTTCTGCATCGT	GACAGAGGAGAAGCTGTTTG	68	17	200	20	58.444	59.584	52.941	50.000	TCCCTCTTCTGCATCGTCCTCGGGCTCCGGCTTGGTGCTCACGCACACAGGAAAGTCCTTCAGCTTCTCCTGCTGTTCCAGCTGCTCTCTCTTGCTGATGGACAAGGGGGCATCAAACAGCTTCTCCTCTGTC	PASS	exon_chr1:17605-17742	exon_chr1:17232-17368	spliced
circ0	chr1	16606	17055	0	TGGCACATAGAAGTAGTTCTC	CCACTCCCACCCTCAA	123	21	228	16	58.233	58.477	42.857	62.500	TGGCACATAGAAGTAGTTCTCTGGGACCTGAAGGAGACGCGCTGCTGCTGCTGTCGTCCTGCCTGGCGCCTTGGCCTACAGGGGCCGCGGTTGAGGGTGGGAGTGG	PASS	exon_chr1:16905-17055	exon_chr1:16606-16756	spliced
