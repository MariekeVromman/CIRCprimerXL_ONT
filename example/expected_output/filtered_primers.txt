circ_ID	chr	start	end	primer_ID	FWD_primer	REV_primer	FWD_pos	FWD_length	REV_pos	REV_length	FWD_TM	REV_TM	FWD_GC	REV_GC	amplicon	PASS
circ1	chr1	17232	17742	0	CTGATGTTGCTGGGAAGAC	GACAGAGGAGAAGCTGTTTG	56	19	214	20	59.531	59.584	52.632	50.000	CTGATGTTGCTGGGAAGACCCCCAAGTCCCTCTTCTGCATCGTCCTCGGGCTCCGGCTTGGTGCTCACGCACACAGGAAAGTCCTTCAGCTTCTCCTGCTGTTCCAGCTGCTCTCTCTTGCTGATGGACAAGGGGGCATCAAACAGCTTCTCCTCTGTC	PASS
circ0	chr1	16606	17055	0	TGGCACATAGAAGTAGTTCTC	CCACTCCCACCCTCAA	124	21	229	16	58.233	58.477	42.857	62.500	TGGCACATAGAAGTAGTTCTCTGGGACCTGAAGGAGACGCGCTGCTGCTGCTGTCGTCCTGCCTGGCGCCTTGGCCTACAGGGGCCGCGGTTGAGGGTGGGAGTGG	PASS
circ2	chr1	17525	18061	0	CGTGAAGATGGAGCCATATT	CTTCAACACCACCGAGAAC	88	20	251	19	58.850	59.762	45.000	52.632	CGTGAAGATGGAGCCATATTCCTGCAGGCGCCCTGGAGCAGGGTACTTGGCACTGGAGAACACCTGTGTCTGATGCCCTGGGTCCCCACTAAGCCAGGCCGGGCCTCCCGCCCACACCCCTCGGCCCTGCCCTCTGGCCATACAGGTTCTCGGTGGTGTTGAAG	PASS
